public/js/pimcore/document/document.js

```diff
@@ -234,6 +234,10 @@ pimcore.document.document = Class.create(pimcore.element.abstract, {
                     this.toolbarButtons.save.hide();
                 }
 
+                if (this.toolbarButtons.save && this.toolbarButtons.publish) {
```

Comment:
when i create a new document, the "save & publish" button is missing. Is it intended?

Reply:
Yes, it's indented. We combined save & publish with the normal save button. You only see "save & publish" for published elements otherwise it's located under the save button, now.